reinforcement-learning-games/zombsole-gym-example/train_dqn.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+'''
+Created: 2021-07-20
+Author: Justin Smith (jvstinian@gmail.com)
+'''
+import os
+import argparse
+# import tensorflow as tf
+# from config import DEMO
+# from task import Task
+# from dpg import DPG
+from dqn.q_learning import DQN
+from dqn.config import DEMO, DEMO_CNN, ZOMBSOLE_MLP
+# from dqn.environment import new_atari_game, new_demo
+import gym
+from zombsole.gym_env import ZombsoleGymEnv, ZombsoleGymEnvDiscreteAction
+from zombsole.renderer import OpencvRenderer
+from gym.envs.registration import registry, register
+import tensorflow.compat.v1 as tf
+tf.disable_v2_behavior()
+
+register(
+    id='Zombsole-v0', 
+    # entry_point='zombsole.gym_env:ZombsoleGymEnv', 
+    entry_point='zombsole.gym_env:ZombsoleGymEnvDiscreteAction', 
+    max_episode_steps=1000,
+    kwargs={
+        'rules_name': 'extermination',
+        # 'player_names': ['sniper', 'sniper', 'sniper', 'terminator'],
+        'player_names': [],
+        'map_name': 'bridge',
+        'agent_id': 0, # 'agent00',
+        'initial_zombies': 5,
+        'minimum_zombies': 0,
+        'debug': False
+    }
+)
+
+def delete_dir(path):
```

Comment:
"delete" in the name of the method is misleading here, as the directory is deleted and then recreated with a `mkdir` call.  Perhaps this should be called "truncate_dir"?